packages/protocol/test/common/integration.ts

```diff
@@ -11,6 +11,7 @@ import {
   ReserveInstance,
   StableTokenInstance,
 } from 'types'
+const soliditySha3 = new (require('web3'))().utils.soliditySha3
```

Comment:
can't we use `Web3.utils.soliditySha3` and also avoid the `require`?

Reply:
+1, we should be able to use the `web3` provided to us by truffle